py/test/selenium/test_remote_connection.py

```diff
@@ -0,0 +1,66 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
```

Comment:
I don't think this is the correct location for this test... other than moving into webdriver, we'd also need to make sure the test is actually run.

@davehunt